Helper/Api/Orders.php

```diff
@@ -934,4 +942,48 @@ private function getTaxRateId(Item $item, ?int $taxFreeId, array $rates): ?int
 
         return $taxRateId;
     }
+
+    /**
+     * @param int|float|string $price
+     * @return Money
+     */
+    private function getBrickMoneyPrice($price): Money
+    {
+        return Money::of($price, $this->order->getStoreCurrencyCode());
+    }
+
+    /**
+     * @param int|float|string $price
+     * @return float
+     */
+    private function getPriceValueForPayload($price): float
+    {
+        return $this->getBrickMoneyPrice($price)->getAmount()->toFloat();
+    }
+
+    /**
+     * @param Money $brickMoneyObj
+     * @return float
+     */
+    private function getPriceByBrickMoneyObj(Money $brickMoneyObj): float
+    {
+        return $brickMoneyObj->getAmount()->toFloat();
+    }
+
+    /**
+     * @param Money $firstOperand
+     * @param Money $secondOperand
+     * @param string $operation
+     * @return Money
+     * @throws \Brick\Money\Exception\MoneyMismatchException
+     */
+    private function calculateBrickMoneyTotal(Money $firstOperand, Money $secondOperand, string $operation): Money
+    {
+        if ($operation == 'plus') {
+            return $firstOperand->plus($secondOperand);
+        }
+        if ($operation == 'minus') {
+            return $firstOperand->minus($secondOperand);
+        }
+    }
```

Comment:
I would remove fully this function and just use `$firstOperand->plus/minus($secondOperand)` where it's used.  It only makes code more complicated

Reply:
function is still there